src/coreclr/vm/nativelibrary.cpp

```diff
@@ -137,7 +138,19 @@ namespace
 
         void SetMessage(LPCSTR message)
         {
+#ifdef TARGET_UNIX
+            //Append dlerror() messages
+            SString new_message = SString(SString::Utf8, message);
+            SString::Iterator i = m_message.Begin();
+            if (!m_message.Find(i, new_message))
+            {
+                m_message += SString(SString::Utf8, " ");
```

Comment:
Is it intentional to have a space at the start of each line?

Reply:
Yes. I thought it's better looking. If you mind, I can remove it.